components/form-builder/app/responses/Dialogs/ConfirmDialog.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from "react";
```

Comment:
Copy+paste from DownloadTableDialog and removes anything related to the ReportDialog